rust/hook-common/src/kafka_messages/app_metrics2.rs

```diff
@@ -0,0 +1,122 @@
+use chrono::{DateTime, Utc};
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+
+use super::{deserialize_datetime, serialize_datetime};
+
+#[derive(Deserialize, Serialize, Debug, PartialEq, Clone)]
+pub enum Source {
+    Hoghooks,
+}
+
+#[derive(Deserialize, Serialize, Debug, PartialEq, Clone)]
+pub enum Kind {
+    Success,
+    Failure,
+}
+
+#[derive(Deserialize, Serialize, Debug, PartialEq, Clone)]
+pub struct AppMetric2 {
+    pub team_id: u32,
+    #[serde(
```

Comment:
Yeah, I made them match the payload we send from Node -> Clickhouse, and since they've been working I will defer any experimentation there for another time/PR.

But I don't disagree!